### DISCUSSION TITLE
How to define model and add layer like keras using C++

### DISCUSSION THREAD

**Discussion by bitnick10** [flashlight/flashlight#656] [Q&A] [answered]
```
model = Sequential()
model.add(Dense(32))
model.add(Dense(5))
model.compile(loss='mse',optimizer='Adam')
```
How to translate those code to C++ simply like keras?

**Comment by tlikhomanenko** [comment, marked as answer]
Please have a look at the example here https://github.com/flashlight/flashlight/blob/master/flashlight/fl/examples/Classification.cpp
